composer.json

```diff
@@ -1,64 +1,84 @@
 {
     "name": "openconext/openconext-profile",
     "license": "Apache-2.0",
+    "description": "OpenConext Profile",
     "type": "project",
     "autoload": {
         "psr-4": {
-            "": "src/"
-        },
-        "classmap": [
-            "src/Kernel.php"
-        ]
+            "OpenConext\\": "src/OpenConext"
+        }
     },
     "require": {
-        "php": "^7.2",
-        "beberlei/assert": "^2.4",
-        "doctrine/annotations": "^1.13",
-        "guzzlehttp/guzzle": "^6.1",
-        "incenteev/composer-parameter-handler": "~2.0",
-        "jms/translation-bundle": "^1.1",
+        "php": "^8.2",
+        "ext-json": "*",
+        "beberlei/assert": "^3.0",
+        "guzzlehttp/guzzle": "^7.0",
+        "incenteev/composer-parameter-handler": "^2.2",
+        "jms/translation-bundle": "^2.0",
         "nelmio/security-bundle": "^2.10",
-        "openconext/monitor-bundle": "^3.1",
-        "sensio/framework-extra-bundle": "^5.5",
-        "surfnet/stepup-saml-bundle": "4.4.*",
-        "symfony/expression-language": "4.4.*",
-        "symfony/flex": "^1.18",
-        "symfony/form": "4.4.*",
-        "symfony/mailer": "4.4.*",
+        "openconext/monitor-bundle": "^4.0",
+        "surfnet/stepup-saml-bundle": "^6.0.18",
+        "symfony/expression-language": "6.4.*",
+        "symfony/flex": "^2",
+        "symfony/form": "6.4.*",
+        "symfony/mailer": "6.4.*",
         "symfony/monolog-bundle": "^3.7",
-        "symfony/security-bundle": "4.4.*",
-        "symfony/translation": "4.4.*",
-        "symfony/twig-bundle": "4.4.*",
-        "symfony/web-profiler-bundle": "4.4.*",
-        "symfony/webpack-encore-bundle": "^1.5",
-        "twig/extensions": "^1.5",
-        "twig/extra-bundle": "^2",
-        "twig/twig": "^2"
+        "symfony/runtime": "6.4.*",
+        "symfony/security-bundle": "6.4.*",
+        "symfony/translation": "6.4.*",
+        "symfony/twig-bridge": "6.4.*",
+        "symfony/twig-bundle": "6.4.*",
+        "symfony/web-profiler-bundle": "6.4.*",
+        "symfony/webpack-encore-bundle": "^2.0",
+        "twig/extra-bundle": "^3.8",
+        "twig/intl-extra": "^3.8",
+        "twig/twig": "^3"
     },
     "require-dev": {
-        "jakub-onderka/php-parallel-lint": "^0.9.2",
-        "malukenho/docheader": "^0.1.6",
-        "matthiasnoback/symfony-config-test": "^4.2",
-        "mockery/mockery": "^0.9.4",
-        "overtrue/phplint": "^2.0",
-        "phpdocumentor/reflection-docblock": "3.3.*",
+        "jakub-onderka/php-parallel-lint": "^1.0",
+        "malukenho/docheader": "^1.0",
+        "matthiasnoback/symfony-config-test": "^5.0",
+        "mockery/mockery": "^1.0.0",
```

Comment:
I see your reasoning, and I agree to an extent. The real bonus is that we are losing a dev dependency. On the other hand, Mockery is so widely used that it will very unlikely pose a threat in the future.